modules/mongodb/src/main/java/org/testcontainers/containers/MongoDbContainer.java

```diff
@@ -0,0 +1,142 @@
+package org.testcontainers.containers;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import lombok.NonNull;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import java.io.IOException;
+
+/**
+ * Constructs a single node MongoDB replica set for testing transactions.
+ * <p>To construct a multi-node MongoDB cluster, consider the <a href="https://github.com/silaev/mongodb-replica-set/">mongodb-replica-set project on GitHub</a>
+ * <p>Tested on a Mongo DB version 4.0.10+ (that is the default version if not specified).
+ *
+ * @author Konstantin Silaev on 9/30/2019
```

Comment:
```suggestion
 * @author Konstantin Silaev on 9/30/2019
```

we're not using the `@author` tags

Reply:
done

Reply:
LGTM